05_Customizing_OrbitDB/07_Implementing_a_custom_AccessController.md

```diff
@@ -0,0 +1,18 @@
+# Implementing a custom Access controllers
+And after these things, let us now consider
+how we might implement a custom access controller.
+
+Starting with the rules, that we want
+to implement:
+- The creator should be the only one to upload notes pieces.
+- Anyone can comment by default.
+- The author of a comment and the creator can delete a comment.
+- The creator can ban somebody from commenting on the entire database.
+
+## Implementation
+Similarly to the `Store`s,
+Access Controllers are implemented
+using inherited classes.
+Let's implement a `NotesAccessController.js`
+file with these Isomorphic bookends:
+```js
```

Comment:
This PR is still WIP, right? Should probably marked as such.

Reply:
It is no longer. Everything after the Conclusion.md is not yet finished.
But there are 6 chapters done, which should be reviewed and merged. 
While I am still working and later providing an Appendix on the Access Controllers.

But the ACLs are separate from the Store & Index.
So it shouldn't be part of the proper Tutorial.

Reply:
I know. But I don't feel, that I have enough expertise about ACs to continue writing this chapter.

Reply:
At least delete line 18 so it doesnt leave a dangling code block